test/markup/cpp/initializers.expect.txt

```diff
@@ -0,0 +1,2 @@
+<span class="hljs-function">string <span class="hljs-title">a</span><span class="hljs-params">(<span class="hljs-string">&quot;hello&quot;</span>)</span>, <span class="hljs-title">b</span><span class="hljs-params">(<span class="hljs-string">&quot;world&quot;</span>)</span></span>;
+<span class="hljs-function">vector&lt;<span class="hljs-keyword">int</span>&gt; <span class="hljs-title">u</span><span class="hljs-params">(<span class="hljs-number">1</span>)</span>, <span class="hljs-title">v</span><span class="hljs-params">(<span class="hljs-number">2</span>)</span></span>;
```

Comment:
Why do you think it is declaring four functions? To me I think it's initializing four variables.

Reply:
If you have found a potential problem please open a new issue rather than commenting on a 3 year old issue.